instruments/nas/client_test.go

```diff
@@ -329,6 +329,161 @@ func TestGet(t *testing.T) {
 	}
 }
 
+func TestClientGetBankAccountFieldFormatting(t *testing.T) {
+	var (
+		httpMetadata = common.HttpMetadata{
+			Status:     "200 OK",
+			StatusCode: http.StatusOK,
+		}
+
+		allowOption = InstrumentSectionFieldAllowedOption{
+			Id:      "1234",
+			Display: "1234",
+		}
+
+		dependencies = InstrumentSectionFieldDependencies{
+			FieldId: "1234",
+			Value:   "1234",
+		}
+
+		sectionField = InstrumentSectionField{
+			Id:              "1234",
+			Section:         "1234",
+			Display:         "1234",
+			HelpText:        "1234",
+			Type:            "1234",
+			Required:        true,
+			ValidationRegex: "1234",
+			MinLength:       0,
+			MaxLength:       1000,
+			AllowedOptions:  []InstrumentSectionFieldAllowedOption{allowOption},
+			Dependencies:    []InstrumentSectionFieldDependencies{dependencies},
+		}
+
+		section = InstrumentSection{
+			Name:   "name",
+			Fields: []InstrumentSectionField{sectionField},
+		}
+
+		query = QueryBankAccountFormatting{
+			AccountHolderType: common.Individual,
+			PaymentNetwork:    common.Ach,
+		}
+
+		response = GetBankAccountFieldFormattingResponse{
+			HttpMetadata: httpMetadata,
+			Sections:     []InstrumentSection{section},
+		}
+	)
+
+	cases := []struct {
+		name             string
+		country          string
+		currency         string
+		query            QueryBankAccountFormatting
+		getAuthorization func(*mock.Mock) mock.Call
+		apiGet           func(*mock.Mock) mock.Call
+		checker          func(*GetBankAccountFieldFormattingResponse, error)
+	}{
+		{
+			name:     "when get back accounts validation",
```

Comment:
This would be better like "When fetching valid bank account validations then it returns validations"

Reply:
👍